File: test/python/unittest-to-pytest/unconvertible/UNCONV-002.expected.py
def test_cleanup():
    # HAMLET-TODO [UNCONVERTIBLE-ADDCLEANUP]: self.addCleanup has no direct pytest equivalent
    # Original: self.addCleanup(print, "cleanup")
    # Manual action required: Use a fixture with yield or request.addfinalizer
    self.addCleanup(print, "cleanup")
    assert True

File: test/java/testng-to-junit5/unconvertible/UNCONV-002.expected.java
import org.junit.jupiter.api.Test;

public class PriorityTest {
    // HAMLET-TODO [UNCONVERTIBLE-PRIORITY]: TestNG priority has no direct JUnit 5 equivalent
    // Original: @Test(priority = 1)
    // Manual action required: Use @Order annotation with @TestMethodOrder(OrderAnnotation.class)
    @Test(priority = 1)
    public void testFirst() {
        assert true;
    }

    // HAMLET-TODO [UNCONVERTIBLE-PRIORITY]: TestNG priority has no direct JUnit 5 equivalent
    // Original: @Test(priority = 2)
    // Manual action required: Use @Order annotation with @TestMethodOrder(OrderAnnotation.class)
    @Test(priority = 2)
    public void testSecond() {
        assert true;
    }
}
